src/io/flutter/logging/FlutterLogTree.java

```diff
@@ -411,6 +413,9 @@ void update() {
 
       reload(getRoot());
       treeTable.updateUI();
+      // Preserve / restore selection state.
+      treeTable.getTree().setSelectionRows(treeTable.getSelectedRows());
```

Comment:
I'm a little surprised this is line is required.

Reply:
Me too.   There's something about the UI update that does not preserve the selection.  I tried a bunch of different things and this is the only thing that worked.